documentation/src/content/docs/en/essentials/manage-states.mdx

```diff
@@ -0,0 +1,240 @@
+---
+title: State Management in effector
+description: How state works in effector and how to manage it
+---
+
+import Tabs from "../../../../components/Tabs/Tabs.astro";
+import TabItem from "../../../../components/Tabs/TabItem.astro";
+
+# State Management
+
+State in effector is managed through stores - special objects that hold values and update them when receiving events. Stores are created using the [createStore](/api/effector/createStore) function.
+
+## Store Creation (#store-creation)
+
+```ts
+import { createStore } from "effector";
+
+// Create store with initial value
+const $counter = createStore(0);
+// with explicit typing
+const $user = createStore<{ name: "Bob"; age: 25 } | null>(null);
+const $posts = createStore<Post[]>([]);
+```
+
+:::tip{title="Store naming"}
+In effector it's conventional to use the `$` prefix for stores. This helps distinguish them from other entities and improves code readability.
+:::
+
+## Reading Values (#read-store-value)
+
+There are several ways to get the current value of a store:
```

Comment:
I think it is better to reorder this variants.
`getState` should be as the last option:

1. useUnit
2. .watch() — only for debug or integration needs
3. .getState() — only for integration needs